ecosystem/sep-0024.md

```diff
@@ -87,6 +87,50 @@ In order for browsers-based wallets to validate the CORS headers, as [specified
 
 This protocol involves the transfer of value, and so HTTPS is required for all endpoints for security.  Wallets and anchors should refuse to interact with any insecure HTTP endpoints.
 
+## Callback signature
```

Comment:
What should implementers do, such as wallets, if they are currently integrated with a server that doesn't provide the callback? We should really be stating what a wallet should do in this case.

Reply:
Do you mean what should wallets do if receiving a callback from an Anchor **that does not provide** a signature?

Reply:
Yup. It might be important, given that this is an active SEP with many existing implementers, for us to note for clients that they should only expect a callback signature from anchors that send callback signatures.

The `/info` endpoint has a list of `features`, and we could include a boolean there for this so that clients can find out if the server supports it.

We should probably do the same thing with the other SEPs that are also adopting this.

cc @JakeUrban

Reply:
I am not sure we should design the ability to not sign the callback as it may slow down the implementation. 
Since there is a real security risk of trusting callbacks without signature, I'd rather say that wallets should not trust a callback that is not signed. 
Now the wallet can still choose to deviate from this spec and have a period of time where the unsigned callback is trusted until the Anchor implement the changes.